src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1517,8 +1517,6 @@ void MavlinkReceiver::fill_thrust(float *thrust_body_array, uint8_t vehicle_type
 	case MAV_TYPE_VTOL_DUOROTOR:
 	case MAV_TYPE_VTOL_QUADROTOR:
 	case MAV_TYPE_VTOL_TILTROTOR:
-	case MAV_TYPE_VTOL_RESERVED2:
-	case MAV_TYPE_VTOL_RESERVED3:
 	case MAV_TYPE_VTOL_RESERVED4:
```

Comment:
IMO `MAV_TYPE_VTOL_RESERVED4` and `MAV_TYPE_VTOL_RESERVED5` should be removed here. But this is opinion. Would defer to @julianoes

Reply:
I would probably leave RESERVED4 and RESERVED5 as is but no strong opinion.

Reply:
I would need to check the discussion back in the Mavlink repository PR, but I left them as is :hand: